### DISCUSSION TITLE
How to copy the notes content, without opening the note

### DISCUSSION THREAD

**Discussion by piyushdongre-dot-com** [chrisgrieser/shimmering-obsidian#188] [Q&A] [answered]
Hi,

I want to copy the content of the note without opening it. My use case is I store snippets in Obsidian, and whenever I am coding in some IDE, I want to quickly copy a snippet from vault to  clipboard quickly. Currently, I use the Codespace.app for this.

Thanks

**Comment by chrisgrieser** [comment, marked as answer]
You can do that by triggering a file action (from Alfred itself) on a note. Iirc, there is a file section for copying the content of a file. 

How a file action is triggered and whether the copy action shows up depends on your general Alfred settings, so please refer to the Alfred forum and/or Alfred docs for that, since that isn't specific to this workflow anymore.
